index.html

```diff
@@ -204,55 +204,1951 @@ <h3>
+                  <ol>
+                    <li>Let |globalState| be the last item of |globalStack|.
+                    </li>
+                    <li>Let |reportId:octet| be |globalState|[`"reportId"`].
+                    </li>
+                    <li>Let |reportItem:HIDReportItem| be the result of
+                    [=creating a HID report item=] with |item|, |localState|,
+                    |globalState|, |usages|, and |strings|.
+                    </li>
+                    <li>For each |collection:HIDCollectionInfo| of |ancestors|:
+                      <ol>
+                        <li>Let |reports:sequence&lt;HIDCollectionInfo&gt;|
+                        be:<br>
+                          |collection|.{{HIDCollectionInfo/inputReports}} if
+                          |item| has the [=Input tag=],<br>
+                          |collection|.{{HIDCollectionInfo/outputReports}} if
+                          |item| has the [=Output tag=], or<br>
+                          |collection|.{{HIDCollectionInfo/featureReports}} if
+                          |item| has the [=Feature tag=].
+                        </li>
+                        <li>
+                          <p>
+                            If |reports| contains an item with its
+                            {{HIDReportInfo/reportId}} attribute equal to
+                            |reportId|, then let |report| be that item.
+                          </p>
+                          <p>
+                            Otherwise:
+                          </p>
+                          <ol>
+                            <li>Let |report:HIDReportInfo| be a
+                            {{HIDReportInfo}}.
+                            </li>
+                            <li>Set |report|.{{HIDReportInfo/reportId}} to
+                            |reportId|.
+                            </li>
+                            <li>Set |report|.{{HIDReportInfo/items}} to an
+                            empty sequence of {{HIDReportItem}}.
+                            </li>
+                            <li>Append |report| to |reports|.
+                            </li>
+                          </ol>
+                        </li>
+                        <li>Append |item| to |report|.{{HIDReportInfo/items}}.
+                        </li>
+                      </ol>
+                    </li>
+                  </ol>
+                </li>
+                <li>If |item| has the [=Collection tag=]:
+                  <ol>
+                    <li>Let |collection:HIDCollectionInfo| be a
+                    {{HIDCollectionInfo}}.
+                    </li>
+                    <li>Let |usage| be the first item of |usages|.
+                    </li>
+                    <li>If the size of |usages| is greater than 1, remove the
+                    first item of |usages|.
+                    </li>
+                    <li>Set |collection|.{{HIDCollectionInfo/usagePage}} to
```

Comment:
Same here for |collection|,
```suggestion
                    <li>Set |collection|["{{HIDCollectionInfo/usagePage}}"] to
```

Reply:
Done